cluster-scope/overlays/moc/common/alertmanager-main-secret.yaml

```diff
@@ -9,34 +9,194 @@ stringData:
     global:
       resolve_timeout: 5m
     inhibit_rules:
-      - equal:
-          - namespace
-          - alertname
-        source_match:
-          severity: critical
-        target_match_re:
-          severity: warning|info
-      - equal:
-          - namespace
-          - alertname
-        source_match:
-          severity: warning
-        target_match_re:
-          severity: info
+    - source_match:
+        severity: critical
+      target_match_re:
+        severity: warning|info
+      equal:
+      - namespace
+      - alertname
+    - source_match:
+        severity: warning
+      target_match_re:
+        severity: info
+      equal:
+      - namespace
+      - alertname
+    - source_match:
+        alertname: ClusterOperatorDegraded
+      target_match_re:
+        alertname: ClusterOperatorDown
+      equal:
+      - namespace
+      - name
+    - source_match:
+        alertname: KubeNodeNotReady
+      target_match_re:
+        alertname: KubeNodeUnreachable
+      equal:
+      - node
+      - instance
+    - source_match:
+        alertname: KubeNodeUnreachable
+      target_match_re:
+        alertname: SDNPodNotReady|TargetDown
+    - source_match:
+        alertname: KubeNodeNotReady
+      target_match_re:
+        alertname: KubeDaemonSetRolloutStuck|KubeDaemonSetMisScheduled|KubeDeploymentReplicasMismatch|KubeStatefulSetReplicasMismatch|KubePodNotReady
+      equal:
+      - instance
+    - source_match:
+        alertname: KubeDeploymentReplicasMismatch
+      target_match_re:
+        alertname: KubePodNotReady|KubePodCrashLooping
+      equal:
+      - namespace
+    - source_match:
+        alertname: ElasticsearchOperatorCSVNotSuccessful
+      target_match_re:
+        alertname: ElasticsearchClusterNotHealthy
+      equal:
+      - dummylabel
     receivers:
-      - name: Default
+      - name: Watchdog
+      - name: "null"
+      - name: Github
         webhook_configs:
           - url: 'http://github-receiver-service.opf-alertreceiver.svc.cluster.local:9393/v1/receiver'
-      - name: Watchdog
     route:
+      receiver: "null"
       group_by:
         - namespace
         - alertname
-      group_interval: 5m
-      group_wait: 30s
-      receiver: Default
-      repeat_interval: 12h
       routes:
-        - match:
+        - receiver: "null"
+          group_by:
+            - namespace
+            - severity
+          continue: true
+          routes:
+            # Alerts we do not want to receive
```

Comment:
Should the `UpdateAvailable` alert be here?

Reply:
let's add it

Reply:
added routing of UpdateAvailable to "null"